packages/babel-plugin-minify-simplify/README.md

```diff
@@ -4,42 +4,46 @@ This plugin will transform code in mainly two ways:
 
 1. Reduce as much statements as possible into expressions
 
-**In**
-```js
-function foo() {
-  if (x) a();
-}
-function foo2() {
-  if (x) a();
-  else b();
-}
-```
-
-**Out**
-```js
-function foo() {
-  x && a();
-}
-function foo2() {
-  x ? a() : b();
-}
-```
+    **In**
+
+    ```js
+    function foo() {
+      if (x) a();
+    }
+    function foo2() {
+      if (x) a();
+      else b();
+    }
+    ```
+
```

Comment:
Identation with code blocks seems to no work. Could you remove them ?

![screenshot from 2016-12-04 18-36-22](https://cloud.githubusercontent.com/assets/1493671/20867953/96111d7a-ba50-11e6-8c1f-8ab35329ff67.png)